lib/validate_style.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+var validateConstants = require('./validate/validate_constants');
+var validate = require('./validate/validate');
+var latestStyleSpec = require('../reference/latest.min');
+var parseStyle = require('./parse_style');
```

Comment:
Also stuff line `require('mapbox-gl-style-spec/lib/validate/latest')` is part of the public API, and although semver in this module is based on the _style spec_ rather than the module API, I'd like to also avoid backwards incompatible changes to the module API outside of major style spec revisions if possible.